src/pudl/package_data/eia923/column_maps/financial_information.csv

```diff
@@ -0,0 +1,31 @@
+year_index,2008,2009,2010,2011,2012,2013,2014,2015,2016,2017,2018,2019,2020,2021,2022
+report_year,year,year,year,year,year,year,year,year,year,year,year,year,year,year,year
+plant_id_eia,plant_id,plant_id,plant_id,plant_id,plant_id,plant_id,plant_id,plant_id,plant_id,plant_id,plant_id,plant_id,plant_id,plant_id,plant_id
+fly_ash_collection_operation_maintenance_expense_thousand_dollars,fly_ash_collection_o_m_expense,fly_ash_collection_o_m_expense,fly_ash_collection_o_m_expense,fly_ash_collection_o_and_m_expense,fly_ash_collection_o_and_m_expense,fly_ash_collection_o_and_m_expense_thousand_dollars,fly_ash_collection_o_and_m_expense_thousand_dollars,fly_ash_collection_o_and_m_expense,fly_ash_collection_o_and_m_expense,fly_ash_collection_o_and_m_expense,fly_ash_collection_o_and_m_expense,fly_ash_collection_o_and_m_expense,fly_ash_collection_o_and_m_expense,fly_ash_collection_o_and_m_expense,fly_ash_collection_o_and_m_expense
+bottom_ash_collection_operation_maintenance_expense_thousand_dollars,bottom_ash_collection_o_m_expense,bottom_ash_collection_o_m_expense,bottom_ash_collection_o_m_expense,bottom_ash_collection_o_and_m_expense,bottom_ash_collection_o_and_m_expense,bottom_ash_collection_o_and_m_expense_thousand_dollars,bottom_ash_collection_o_and_m_expense_thousand_dollars,bottom_ash_collection_o_and_m_expense,bottom_ash_collection_o_and_m_expense,bottom_ash_collection_o_and_m_expense,bottom_ash_collection_o_and_m_expense,bottom_ash_collection_o_and_m_expense,bottom_ash_collection_o_and_m_expense,bottom_ash_collection_o_and_m_expense,bottom_ash_collection_o_and_m_expense
```

Comment:
In other tables we've referred to operation and maintenance expenses with the collective `_opex` suffix/prefix. Don't have a strong preference as to which one we use, just wanted to flag for consistency. Maybe @cmgosnell has thoughts here.

Reply:
in the FERC1 plant tables we pretty exclusively use `opex_` and `capex_` as prefixes for operational expenses and . Also see this [abbreviations table](fly_ash_collection_operation_maintenance_expense_thousand_dollars). I think we've mostly walked away from using `epxns` in lieu of `opex`.

if we want to be consistent with ferc1 i think this guy would probably be opex_bottom_ash_collection_maintenance_thousand_dollars`